### PR TITLE
MMDP-464 Implement manage beneficiary type dropdown

### PR DESCRIPTION
**Pull request by zaabu**
**What does this PR do?**

This PR implements CRUD operations for the  beneficiary type dropdown

**Description of Task to be completed?**

As an administrator, I want to be able to manage options in the 'Beneficiary Type' drop down

**How should this be manually tested?**

Git clone https://github.com/andela/mmdp_site
Check out  ft-beneficiary-type-MMDP-464 branch using git.
Run yarn run: server to start the server.

**Any background context you want to provide?**

None

**What are the relevant Jira issues?**

[MMDP-464](https://viisaus.atlassian.net/browse/MMDP-464)

### CODE REVIEW

File: client/store/sagas/dropdowns/beneficiaryTypes.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    const response = yield call(api.dropdowns.beneficiaryType.create, payload);
+    yield put(actions.addTypesSuccess());
+    yield put(actions.fetchTypes());
+    toastr.success(response.data.message);
+  } catch (error) {
+    yield put(actions.addTypesFailure({}));
+    const message = error.response
+      ? error.response.data.message
+      : 'Error adding types';
+    toastr.warning(message);
```
Comment (eduhmik) [inline client/store/sagas/dropdowns/beneficiaryTypes.js]: Consider using `yield call(toastr.warning, message)` as @p8ul proposed earlier yesterday. Do this for all other toasters.

Comment (eduhmik) [reply client/store/sagas/dropdowns/beneficiaryTypes.js]: Other than this, tested your PR manually and everything works fine. When you have fixed these few changes just ping me to approve your PR.

File: client/assets/styles/pages/_dropdown.scss [hunk 2]
```diff
@@ -4,7 +4,7 @@
         background: $cool-blue;
         border-radius: 50%;
         height: 34px;
-        width: 107px;
+        width: 7%;
```
Comment (Sulaiman-Mozes) [inline client/assets/styles/pages/_dropdown.scss]: image
 
The CSS class you modified breaks the delete button for dropdowns with three fields. Could kindly fix this.

Comment (zaabu) [reply client/assets/styles/pages/_dropdown.scss]: Alright,on it.

**Pull request merged**